streams/src/main/java/org/apache/kafka/streams/kstream/Window.java

```diff
@@ -22,8 +22,8 @@
  */
 public abstract class Window {
 
-    private long start;
-    private long end;
+    protected long start;
```

Comment:
Can we make these fields final? I know they weren't before, but as you are changing this